debugutils/aggregator.go

```diff
@@ -0,0 +1,109 @@
+package debugutils
+
+import (
+	"io"
+	"path/filepath"
+
+	"github.com/solo-io/go-utils/errors"
+	"github.com/solo-io/go-utils/installutils/helmchart"
+	"github.com/solo-io/go-utils/tarutils"
+	"github.com/spf13/afero"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	aggregatorName = "aggregator"
+)
+
+type Aggregator struct {
+	resourceCollector ResourceCollector
+	logCollector      LogCollector
+	storageClient     StorageClient
+	fs                afero.Fs
+
```

Comment:
I would leave a comment here indicating it's the root directory for the collected files on `fs`